pkg/kubernetes/README.md

```diff
@@ -12,7 +12,8 @@ This sets up a single machine kubernetes master and minion:
 Now in order to support the v1beta3 API, we need to build kubernetes
 from source or use a version later than v0.10.0:
 
-    $ git clone git@github.com:GoogleCloudPlatform/kubernetes.git
+    $ sudo yum install golang
+    $ git clone git://github.com/GoogleCloudPlatform/kubernetes.git
```

Comment:
The SSH variant doesn't work when you're trying to do this on nodes that you don't have your SSH keys on. The git variant works everywhere. Unless you have push access to the kubernetes repo, there's really no difference between the two at all.